pkg/limayaml/limayaml.go

```diff
@@ -75,6 +75,11 @@ type File struct {
 	Digest   digest.Digest `yaml:"digest,omitempty" json:"digest,omitempty"`
 }
 
+type Archive struct {
+	File
+	Version string `yaml:"version,omitempty" json:"version,omitempty"`
```

Comment:
```suggestion
	NerdctlVersion string `yaml:"nerdctlVersion,omitempty" json:"nerdctlVersion,omitempty"`
```

Also wondering if this can be just parsed from the URL

Reply:
It could, but then the URL was obfuscated in the cache:

`--nerdctl-archive /home/anders/.cache/lima/download/by-url-sha256/a23acacc818a76fadb89c79cc79548adbdac0942bf562bfc87463861561c824b/data`